chatroom/src/components/Index.jsx

```diff
@@ -0,0 +1,126 @@
+                <button type="submit" class="btn btn-primary">
+                  Login
+                </button>
+              </div>
+            </div>
+            <div class="d-flex justify-content-center">
+              <input
+                class="form-control col-md-3 col-sm-4 col-6 my-3"
+                type="search"
+                placeholder="Search"
+                aria-label="Search"
+              ></input>
+              <i class="fa fa-search mx-2 my-4" aria-hidden="true"></i>
+            </div>
+          </form>
+        </div>
+      </div>
+      <div class="card-group justify-content-center align-items-center">
+        <div class="row my-4">
+          <div class="col-sm-6 col-md-4">
+            <SingleCard
+              title="Chess"
+              text="This is a chatroom for chess enthusiasts. 
+            Hope you'll find openings better than Danish here. :)"
+            />
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <SingleCard
+              title="Music"
+              text="How about something unusual yet prominent?
+            Flaunt your music skills here with people from around the world."
+            />
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <SingleCard
+              title="Movies"
+              text="From timeless classics to drama to crime, this
+            is your place if you've considered movies as an integral part of your daily life."
+            />
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-sm-6 col-md-4">
+            <SingleCard
+              title="Literature"
+              text="Fill this place with some catchy words...................for now i have to type
+              s."
+            />
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <SingleCard
+              title="Mathematics"
+              text="This is a place for geeks. It's better to stay away. xD blah blah blah..
+              the greatest trick the de"
+            />
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <SingleCard
+              title="Mathematics"
+              text="This is a place for geeks. It's better to stay away. xD blah blah blah..
+              the greatest trick the de"
+            />
+          </div>
+        </div>
+        <div class="row my-4">
+          <div class="col-sm-6 col-md-6">
+            <SingleCard
+              title="Chess"
+              text="This is a chatroom for chess enthusiasts. 
+            Hope you'll find openings better than Danish here. :)"
+            />
+          </div>
+          <div class="col-sm-6 col-md-6">
+            <SingleCard
+              title="Music"
+              text="How about something unusual yet prominent?
+            Flaunt your music skills here with people from around the world."
+            />
+          </div>
+          {
+            data.map((cardDetails)=>{
```

Comment:
You can use a grid or a ul-li element to wrap this SingleCard for easier means of handling the display on the screen.